awx/main/utils/encryption.py

```diff
@@ -63,7 +63,13 @@ def encrypt_field(instance, field_name, ask=False, subfield=None):
     '''
     Return content of the given instance and field name encrypted.
     '''
-    value = getattr(instance, field_name)
+    try:
```

Comment:
@alancoding We had some discussions earlier about doing this using a separate set of functions (decrypt_inputs, encrypt_inputs). After digging into this some more, I do think that extending the existing function is probably the way to go - There's places where we invoke `decrypt_field` on generic objects where the specific model type isn't known (copying is one example). If we added more functions we'd need to test objects for their type to know which set of functions to use. 

I realize that adding a brick of exception handlers to both functions isn't the nicest thing to look at, but the `credential.inputs` case _is_ exceptional so handling them with exceptions seems like a [valid way to deal with this sort of thing in python](https://docs.python.org/3/glossary.html#term-eafp). What do you think?